src/chain_monitor.ts

```diff
@@ -0,0 +1,123 @@
+import { ExtendedAccount } from "./config"
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { Hash, Header } from "@polkadot/types/interfaces/runtime";
+import { GenericExtrinsic } from "@polkadot/types/"
+
+import "@polkadot/api-augment";
+import "@polkadot/types-augment";
+
+const DEBUG = false;
+
+export interface ExtrinsicItem {
+    index: number,
+    section: string,
+    method: string,
+    account: ExtendedAccount,
```

Comment:
who is account? what if it is an unsigned extrinsic?

Reply:
Unsigned extrinsics now take either the related account from the extrinsic as a string, OR remain undefined if there are no accounts set and the extrinsic is in the extrinsicFilter.